app/models/krikri/activity.rb

```diff
@@ -105,10 +105,20 @@ def to_s
     # Return an Enumerator of URI strings of entities (e.g. aggregations or
     # original records) that pertain to this activity
     #
+    # @param  include_invalidated [Boolean] Whether to include entities that
+    #                                       have been invalidated with
```

Comment:
can we indent these less aggressively? It seems like two spaces on hanging lines should be enough.